### PR TITLE
Add test of finding stationary points

### PR DESCRIPTION
**Pull request by dpsanders**

### CODE REVIEW

File: test/stationary_points.jl [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+ForwardDiff.gradient(f, X::IntervalBox) = ForwardDiff.gradient(f, X.v)
+
+∇f = X -> ForwardDiff.gradient(f, X)
+
+L = 5.0
+X = IntervalBox( (-L..L+1), 2)
+
+@time rts = IntervalRootFinding.roots(∇f, X, Bisection)
+@time rts = IntervalRootFinding.roots(∇f, rts, Newton, 1e-20)
```
Comment (Kolaru) [inline test/stationary_points.jl]: These two `@time` can be removed. I'll add them as benchmark to #105.

Also with minor adaptation Krawczyk method could be checked here too (a bit to involved change for GitHub propose change feature though).

File: test/stationary_points.jl [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+@test length(rts) == 529
```
Comment (Kolaru) [inline test/stationary_points.jl]:
```suggestion
@test length(rts) == 529
@test all(isunique(rts))
```

Checking that the method guarantee unicity can not hurt.

Comment (dpsanders) [reply test/stationary_points.jl]: Good points, thanks!

**Comment by Kolaru** [comment]
Superseded by #205

**Pull request closed without merging**